solo-chains/runtime/dancelight/src/bridge_to_ethereum_config.rs

```diff
@@ -140,16 +153,23 @@ impl snowbridge_pallet_system::Config for Runtime {
     #[cfg(feature = "runtime-benchmarks")]
     type Helper = benchmark_helper::EthSystemBenchHelper;
     type DefaultPricingParameters = Parameters;
-    type InboundDeliveryCost = ();
-    //type InboundDeliveryCost = EthereumInboundQueue;
+    type InboundDeliveryCost = EthereumInboundQueue;
```

Comment:
what are the implicatrions of this? are we going to charge for the validators payload?

Reply:
from what I can see it is the price that we can communicate to the SC, so I think its correct, but plz confirm